cli/src/cmd/error.rs

```diff
@@ -30,12 +30,34 @@ pub enum CommandErrorKind {
     Other(String),
 }
 
+impl std::fmt::Display for CommandErrorKind {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self {
+            CommandErrorKind::Io(err) => write!(f, "Io: {:?}", err),
+            CommandErrorKind::UnimplementedCommand => write!(f, "Command unimplemented"),
+            CommandErrorKind::UnimplementedAbstractionLayer => {
+                write!(f, "Abstraction layer unimplemented")
+            }
+            CommandErrorKind::ZipError(err) => {
+                write!(f, "Error unzipping template: {:?}", err)
+            }
+            CommandErrorKind::Other(err) => write!(f, "{}", err),
+        }
+    }
+}
+
 /// An error that can be encountered while executing commands.
 #[derive(Debug)]
 pub struct CommandError {
     kind: CommandErrorKind,
 }
 
+impl std::fmt::Display for CommandError {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{}", self.kind)
+    }
+}
+
 impl From<IoError> for CommandError {
```

Comment:
We could replace impls like these by usage of: https://crates.io/crates/derive_more

Reply:
I would agree, but to do so I would need to move `CommandErrorKind` variants up to the `CommandError` enum itself so they can be derived. Happy to do so if you agree.